addons/l10n_cl/views/report_invoice.xml

```diff
@@ -187,6 +179,32 @@
             <attribute name="t-attf-class">#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'}</attribute>
         </xpath>
 
+        <t t-foreach="o.amount_by_group" t-as="amount_by_group" position="replace">
+            <t t-if="o.l10n_latam_document_type_id.code not in ['39', '41', '110', '111', '112', '34']">
+                <t t-foreach="o.amount_by_group" t-as="amount_by_group">
+                    <tr style="">
+                        <t t-if="len(o.line_ids.filtered(lambda line: line.tax_line_id)) in [0, 1] and o.amount_untaxed == amount_by_group[2]">
+                            <td><span class="text-nowrap" t-esc="amount_by_group[0]"/></td>
+                            <td class="text-right o_price_total">
```

Comment:
This thing is weird and hacky.  Same thing must exist for AR.  (the amount_by_group method was even inherited for this, ...)

Reply:
@Danisan check this: https://github.com/odoo/odoo/blob/84ffb9cd34c9abc76ba807549567b383750bda7c/addons/l10n_latam_invoice_document/models/account_move.py#L223

Reply:
Sorry, by I don't catch what is the change you are asking for. Seems to me that this part of the report is as hacky or tricky as the 
 default account.report_invoice_document similar section. At the end, even when the method _compute_invoice_taxes_by_group is inherited, it serves to compute amount_by_group, and seems that the structure is the same. In AR i did not find a section for this. They only use the latam_taxes_ids in the lines, while here we are skiping to show taxes in the lines, and just at the footer of the document.
Should I use a different thing to show the totals / taxes / subtotals? please tell me.

Reply:
@Danisan What is not working if you just remove all of this?

Reply:
@Danisan let's try removing this section and see if it works